### ISSUE TITLE
Variation in ETRs on capital across financing and tax treatment

### ISSUE THREAD

**Issue by jdebacker** [open-source-economics/plot_concepts#6]
Plot concept: In thinking about business tax reform, it's useful to consider how taxes distort investment choices across business entity form and the type of financing involved.  The plot concept allows one to visualize the extent of those distortions by considering the variation in tax rates on investments with different financing and by corporate and non-corporate entities.  The idea for the plot, which is inspired by Figure 1 in [CBO (2014)](https://www.cbo.gov/sites/default/files/113th-congress-2013-2014/reports/49817-Taxing_Capital_Income_0.pdf), is to use a box plot to represent the range of marginal effective tax rates on investment by tax treatment and financing type.

<img width="557" alt="cbo_etr_boxplot" src="https://cloud.githubusercontent.com/assets/10715924/17746885/483dc33a-6480-11e6-9cc5-4126ccd5e186.png">

Technical notes:
- The Cost of Capital Calculator at https://github.com/open-source-economics/B-Tax can perform calculations from which one could create such a plot.
- While the variation under current law is interesting, one could make the box plot dynamic, with switches to turn on and off certain policies affecting METRs on investments: e.g. bonus depreciation, interest deductibility, or an allowance for corporate equity, among other items. In this case, switches could be used on the plot to turn these policies on or off and the box plot would update accordingly.

**Comment by MattHJensen** [comment]
[This story](http://www.wsj.com/articles/donald-trumps-tax-ideas-could-boost-debt-laden-real-estate-firms-1471542621) in the WSJ used a similar plot: 

![image](https://cloud.githubusercontent.com/assets/8301092/17857686/bc314b58-6850-11e6-91f8-df0a3845b478.png)

**Comment by MattHJensen** [comment]
@jdebacker, do you think it would make sense to make a visualization out of the chart that you posted with these parameters? 

C-Corporation top rates: 
[39.6, **35**, 30, 25, 20, 15, 0]

Depreciation 
[Economic Depreciation, Current Law, no Bonus Depreciation, **Current Law, with Bonus Depreciation**, 100% expensing]

Interest deductibility [**Fully deductible**, Not deductible]

After https://github.com/open-source-economics/Tax-Calculator/pull/868 is merged, we could add:

Pass-Through top rates: 
[**39.6**, 35, 30, 25, 20, 15, 0]

The chart would be exactly as you show it. (The content for the note and source would of course be different).

**Comment by jdebacker** [comment]
@MattHJensen , yes the parameterization you propose would be interesting given the current candidates' proposals.  We could also represent this reform with the bubble plot proposed [here](https://github.com/open-source-economics/plot-concepts/issues/7#issuecomment-241493509)).  @brendancol is working on adding different color for baseline vs reform.
